Dockerfile

```
@@ -1,4 +1,4 @@
-FROM openmicroscopy/omero-web-standalone:latest
+FROM openmicroscopy/omero-web-standalone:5.21.0
```

```suggestion
FROM openmicroscopy/omero-web-standalone:latest
```

as per @jburel's comment - this works fine for me

---

> this works fine for me

Sorry, have to eat my words. It works fine only up to a point. This means that the docker image builds fine, and the docker starts as expected. I am able to connect to demo server. But then:

- if using the latest image, I am not able to watch the changes live as described in this README, for example changing the ``index.html`` file and renaming the ``Export PDF`` button does not appear immediately after refresh
- if using the 5.21.0 image, I am able to watch the changes live, ie. the ``Export PDF`` button manipulation in the ``index.html`` is immediately visible after refresh of the browser 

- if